mirrord-layer/src/socket/ops.rs

```diff
@@ -210,167 +275,36 @@ pub(super) fn connect(sockfd: RawFd, remote_address: SocketAddr) -> HookResult<(
         .then_some(())
         .ok_or_else(|| HookError::BypassedPort(remote_address.port()))?;
```

Comment:
Is this the same as writing`if is_ignored_port(remote_address.port()) {return BypassedPort...}`?

Reply:
Would the code I suggested work though? Or do we prefer chaining methods, stylewise?

Reply:
Your suggestion works. I'm not a fan of early returns like that, and prefer chaining things.

Should I open a discussion to gather feedback from our pals?

Reply:
Yeah lets do it

Reply:
Poll is open, I'll mark this as resolved and wait for feedback (then we can just refactor it later).